README.md

```diff
@@ -39,6 +40,100 @@ Note that politeiawww does not provide HTML output.  It strictly handles the
 JSON REST RPC commands only.  The GUI for politeiawww can be found at:
 https://github.com/decred/politeiagui
 
+## Development 
+
+#### 1. If you haven't install Go, check it out the [docs fot it](https://golang.org/doc/install)
+#### 2. Get this repository properly cloned and installed. 
+#### 3. If you have properly setted your enviroment you will be able to run Politeia's components by simple typing its name (e.g: `politeiawwww`) on your terminal (p.s It won't fully work untill you set its configuration).
+#### 4. Finding your configuration files path: 
+* To make things simpler during development is handy having a configuration file for your components. Specially `politeiad` and `politeiawww` so you don't have to pass the parameters every time you run it. 
+* You may find the configuration files under the path:  
+  - **Mac OS** `/Users/<username>/Library/Application Support/Politeiad/politeiad.conf`
+  - **Windows** `C:\Users\<username>\AppData\Local\Politeiad/politeiad.conf`
+  - **Ubuntu** `~/.politeiad/politeiad.conf`
+
+#### 5. Set the configuration files:
+  
+Add the following configurations:
+
+**politeiad.conf**: 
+
+    rpcuser=user
+    rpcpass=pass
+    testnet=true	
+
+
+**politeiawww.conf**:
+
+    rpchost=127.0.0.1
+    rpcuser=user
+    rpcpass=pass
+    rpccert="/Users/<username>/Library/Application 
+    Support/Politeiad/https.cert"
+    proxy=true
+    testnet=true
+
+**Attention:** The rpccert path is referenced on MacOs path. See step 4 for more OS paths.
+#### 6. Run Politeiad by simple running on your terminal:
+
+    politeiad
+
+#### 7. Download server identity to client:
+
+    politeia -v -testnet -rpchost 127.0.0.1 identity
+
+Accept default path by pressing `enter`.
+
+Result should look something like this:
+
+```
+FQDN       : localhost
+Nick       : politeiad
+Key        : dfd6caacf0bbe5725efc67e703e912c37931b4edbf17122947a1e0fcd9755f6d
+Identity   : 99e748e13d7ecf70ef6b5afa376d692cd7cb4dbb3d26fa83f417d29e44c6bb6c
+Fingerprint: medI4T1+z3Dva1r6N21pLNfLTbs9JvqD9BfSnkTGu2w=
+
+Save to /Users/<username>/Library/Application Support/Politeia/identity.json or ctrl-c to abort
+Identity saved to: /Users/<username>/Library/Application Support/Politeia/identity.json
+```
+
+#### 8. Run Politeiawww by simple running on your terminal:
```

Comment:
I would change the wording to: Start the Politeiawww server by running on your terminal: